src/renderer/actions/install-path.utils.tsx

```diff
@@ -1,7 +1,76 @@
-import settings, { msStoreBasePath, steamBasePath } from 'renderer/rendererSettings';
+import * as fs from 'fs';
+import walk from 'walkdir';
+import * as path from 'path';
+import * as os from 'os';
+import settings from 'renderer/rendererSettings';
 import { Directories } from 'renderer/utils/Directories';
 import { dialog } from '@electron/remote';
-import fs from 'fs';
+import { managedSim, TypeOfSimulator } from 'renderer/utils/SimManager';
+
+const possibleBasePaths = {
```

Comment:
I think this should be typed a bit better, with an explicit mapping: `Record<TypeOfSimulator, { store: string, steam: string }>`